ha/core/event_analyzer/event_analyzerd.py

```diff
@@ -94,9 +98,15 @@ def run(self):
         """
         for watcher in self._watcher_list.keys():
             Log.info(f"Starting watcher {watcher} service for event analyzer.")
+            if self._watcher_list[watcher] == self._iem_watcher:
+                continue
             self._watcher_list[watcher].start()
         Log.info(f"Running the daemon for HA event analyzer with PID {os.getpid()}...")
         while True:
+            component = 'Manager'
+            EventMessage.subscribe(component)
+            iem_alert_message = EventMessage.receive()
```

Comment:
Is the event analyzer not going to receive anything except IEMs at the moment ? Can you please confirm ?